subscription-sender/cmd/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/RinaDish/subscription-sender/internal/app"
+	"github.com/RinaDish/subscription-sender/tools"
+	"github.com/kelseyhightower/envconfig"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func main() {
+	var cfg app.Config
+	err := envconfig.Process("subscription-sender", &cfg)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	config := zap.NewProductionConfig()
+	config.EncoderConfig.EncodeTime = zapcore.TimeEncoderOfLayout(time.RFC3339)
+
+	l, _ := config.Build()
```

Comment:
I don't see what could go wrong in this case, it's not necessary to handle every error in Go. Do you think this is critical to fix? @boredgus

Reply:
In general it is not big deal when you omit error check once in place where chance of fail is extremely low, but in perspective desire to repeat it will grow. 

There is good phrase from The Go Blog article: ["Proper error handling is an essential requirement of good software"](https://go.dev/blog/error-handling-and-go#:~:text=Proper%20error%20handling%20is%20an%20essential%20requirement%20of%20good%20software)

Reply:
@boredgus Also a good [article](https://preslav.me/2023/04/14/golang-error-handling-is-a-form-of-storytelling/) about how to handle only those errors that need to be handled ))

Reply:
As I understood, author of article tells about his approach of describing the error to simplify debugging.

Also phrase from article: ["In Go, the same causes a runtime crash across the entire app. I call that a refreshing way of teaching you to respect error handling and leaving as few opportunities for a runtime panic as possible."](https://preslav.me/2023/04/14/golang-error-handling-is-a-form-of-storytelling/#:~:text=In%20Go%2C%20the%20same%20causes%20a%20runtime%20crash%20across%20the%C2%A0entire%20app.%20I%20call%20that%20a%20refreshing%20way%20of%20teaching%20you%20to%20respect%20error%20handling%20and%20leaving%20as%20few%20opportunities%20for%20a%20runtime%20panic%20as%20possible.) 

🙃 

If you are sure in your opinion, left it as it is. I'm just supporting the discussion)

Reply:
Great discussion, I love it)

Here you have 2 types of errors.
1. Error while building/bootstrapping your app. As for me here you shouldn't let your app start if something goes wrong.
2. Business  errors. This should be handled properly firstly for client of your app.
3. Infrastructure errors. Connection errors, stream errors whatever. This could be also at first point, when you're establishing first connections. Here you should think about behaviour of your application in cases of failure. 

So I will just add panic on error, so i will be confident that all goes right.